autobahn/asyncio/component.py

```diff
@@ -469,8 +469,11 @@ def nicely_exit(signal):
     loop.add_signal_handler(signal.SIGINT, partial(nicely_exit, 'SIGINT'))
     loop.add_signal_handler(signal.SIGTERM, partial(nicely_exit, 'SIGTERM'))
 
+    def done_callback(reactor, arg):
+        reactor.stop()
+
     # returns a future; could run_until_complete() but see below
```

Comment:
very minor nit-pick (and I'm not even sure if we're consistent here already) but asyncio usually calls the reactor "loop"

Reply:
just pushed another commit that changes that.